R/pkg/R/DataFrame.R

```diff
@@ -1465,10 +1464,10 @@ setMethod("dapplyCollect",
 #'
 #' Result
 #' ---------
-#' Model  (Intercept)  Sepal_Width  Petal_Length  Petal_Width
-#' 1        0.699883    0.3303370    0.9455356    -0.1697527
-#' 2        1.895540    0.3868576    0.9083370    -0.6792238
-#' 3        2.351890    0.6548350    0.2375602     0.2521257
+#' Model  Species     (Intercept)  Sepal_Width  Petal_Length  Petal_Width
```

Comment:
This is an external change. I think such an external change is not acceptable after we already introduce it. Right?